packages/core-mobile/app/new/routes/(authenticator)/components/AuthenticatorSetup.tsx

```diff
@@ -0,0 +1,143 @@
+              </View>
+              <View sx={{ flex: 1 }}>
+                <View sx={{ gap: 4 }}>
+                  <Text
+                    sx={{
+                      fontSize: 16,
+                      fontWeight: '500',
+                      lineHeight: 16,
+                      color: colors.$textPrimary
+                    }}>
+                    Copy code
+                  </Text>
+                  <View
+                    sx={{ marginTop: 3, marginRight: 39, marginBottom: 16 }}>
+                    {totpKey ? (
+                      <Text
+                        sx={{
+                          fontSize: 12,
+                          fontWeight: '400',
+                          lineHeight: 15,
+                          color: colors.$textSecondary
+                        }}>
+                        {totpKey}
+                      </Text>
+                    ) : (
+                      <ActivityIndicator
+                        size={16}
+                        color={colors.$textPrimary}
+                        sx={{
+                          alignSelf: 'flex-start'
+                        }}
+                      />
+                    )}
+                  </View>
+                </View>
+                <Separator />
+              </View>
+            </TouchableOpacity>
+
+            <TouchableOpacity
+              onPress={onScanQrCode}
+              sx={{
+                flexDirection: 'row',
+                alignItems: 'center',
+                gap: 16,
+                width: '100%',
+                marginTop: 16
+              }}>
+              <View sx={{ width: 18, alignItems: 'center' }}>
+                <Icons.RecoveryMethod.QrCode color={colors.$textPrimary} />
+              </View>
+              <View sx={{ flex: 1 }}>
+                <View sx={{ gap: 4 }}>
+                  <Text
+                    sx={{
+                      fontSize: 16,
+                      fontWeight: '500',
+                      lineHeight: 16,
+                      color: colors.$textPrimary
+                    }}>
+                    Scan QR code
+                  </Text>
+                  <Text
+                    sx={{
+                      fontSize: 12,
+                      fontWeight: '400',
+                      lineHeight: 15,
+                      color: colors.$textSecondary,
+                      marginTop: 3,
+                      marginRight: 39
+                    }}>
+                    View QR code to scan with your authenticator app
+                  </Text>
+                </View>
+              </View>
+            </TouchableOpacity>
+          </Card>
+        </View>
+        <View sx={{ marginBottom: 36 }}>
+          <Button type="primary" size="large" onPress={onVerifyCode}>
+            Next
+          </Button>
+        </View>
+      </View>
+    </BlurredBarsContentLayout>
+  )
+}
```

Comment:
can we move these components under `(authenticator)/components` to `app/new/components`?